CONTRIBUTION.md

```diff
@@ -74,3 +74,19 @@ storiesOf('Molecules/Top content bar', module)
   .addDecorator(withBodyReset())
   .add('Top content bar - default', () => '<axa-top-content-bar>Some children</axa-top-content-bar>');
 ```
+
+## Release your components
+
+First of all, make sure that you have a npm account under the axa-ch org and that you are logged in.
+Make sure you have installed lerna globally: `npm i -g lerna`
```

Comment:
Installing lerna globally is not needed, thanks to npx.

Reply:
It is cause in the step above we say to do : rm -r node_modules

Reply:
I corrected it with the update. discussed with @LucaMele